standards/code-style.adoc

```diff
@@ -520,26 +526,26 @@ recommendable to use the following order:
 
 Example:
 
-[source, java]
+[source, kotlin]
 ----
-public class MainActivity extends Activity {
+class MainActivity : Activity() {
 
-    private String title;
-    private TextView textViewTitle;
+    private var title: String? = null
+    private val textViewTitle: TextView? = null
 
-    @Override public void onCreate() {
+    fun onCreate() {
         // ...
     }
 
-    public void setTitle(String title) {
-    	this.title = title;
+    fun setTitle(title: String) {
+        this.title = title
     }
 
-    private void setUpView() {
+    private fun setUpView() {
         // ...
     }
 
-    static class AnInnerClass {
+    internal class AnInnerClass {
```

Comment:
The ```inner``` keyword is more appropriate here instead of ```internal```.

Reply:
Updated